src/docs/product/performance/filters-display/index.mdx

```diff
@@ -38,6 +38,23 @@ When you select "All Transactions", the table includes all transactions across a
 
 "Mobile" puts an emphasis on [Mobile Vitals](/product/performance/mobile-vitals/). Note that if you've selected a non-mobile project, this mode also displays all of those transactions.
 
+## Transaction Search
+
+<Note>
+
+This feature is in beta and is only available if your organization is participating in its limited release. Features in beta are still in-progress and may have bugs. We recognize the irony. If you’re interested in participating, [join the waitlist](https://sentry.io/for/performance/#updates-signup).
+
+</Note>
+
+The transaction filter on the performance page has been simplified to support metric enhanced performance. The search bar can only be used to filter by transaction name.
```

Comment:
Maybe helpful to clarify the difference from the regular search bar. Maybe something like "The search bar can only be used to filter by transaction name, rather than to search for transactions matching the query".

Reply:
Also, I wonder if MEP is a marketing-visible feature, in which case it should be capitalized?

Reply:
As suggested by @nanoburd we replaced the MEP with metrics, as metrics has documentation to link to. Also, I agree, and reworded the search bar docs as suggested!